README.md

```diff
@@ -4,7 +4,11 @@
 
 1. Setup env variables
 ```
+// manual
 cp .env.sample .env.local
+
+// generate by script (aws logged in)
+yarn env
```

Comment:
Please split the code block.
Commands in a single code block may looks like that they're all needed